### DISCUSSION TITLE
Create link in tinymce lists only the anchors of the current block

### DISCUSSION THREAD

**Discussion by Vgratioulet** [voidlabs/mosaico#633] [Q&A]
I added the 'anchor' plugin for tinymce and the create link buttons helps by listing the anchors available.
I would like that the list would show the anchors of the whole template.
Do you know how to do that ?
Thanks

**Comment by bago** [comment]
No, sorry I don't know.
I don't think it is a good idea to use anchors in email, so I don't think it worth investigating on this.

**Comment by Vgratioulet** [comment]
ok, thank you, I'll ask the users to think again why we need this !
